dex/testing/loadbot/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -e;
+go build;
+./loadbot -p $@ | tee -i bot.log;
```

Comment:
Nothing that needs changing, but I wanted to note that I was testing like `go run . -p pingpong4 -debug`